src/components/tabbar/icons.js

```diff
@@ -1,6 +1,6 @@
 // @flow
```

Comment:
Why is this file flagged as a generated file??

Reply:
Not sure, there's no .gitattributes file.